src/Libraries/CoreNodeModels/Input/IntegerSlider.cs

```diff
@@ -43,10 +43,10 @@ public override int Value
             set
             {
                 base.Value = value;
-                if (base.Value > Int32.MaxValue)
-                    base.Value = Int32.MaxValue;
-                if (base.Value < Int32.MinValue)
-                    base.Value = Int32.MinValue;
+                if (base.Value >= Int32.MaxValue)
```

Comment:
I assume that logic elsewhere is handling the case that this was originally meant to solve, resetting the value if max or min is exceeded. Is that correct?